README.md

```diff
@@ -11,22 +11,22 @@ It consists of the following sections :
 
 Join Us
 -------
-Send us a mail to lyonytechhub@googlegroups.com or even request to join the group, it is totally free.

```

Comment:
There was a typo in the contact email 😅.
Also, `lyontechhub@googlegroups.com` can be safely replaced by `contact@lyontechhub.org`, right?